backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/Textfield.java

```diff
@@ -96,6 +104,11 @@ public Builder newValueHandler(Function<String, IStatus> newValueHandler) {
             return this;
         }
 
+        public Builder style(TextfieldStyle style) {
+            this.style = style;
```

Comment:
`Objects.requireNonNull` If someone explicitly gives us a value, it has no reason to be `null`

Reply:
Done